ili/utils/ndes_pt.py

```diff
@@ -287,6 +290,12 @@ def load_nde_lampe(
             flow_class = zuko.flows.autoregressive.MAF
         elif model == 'nsf':
             flow_class = zuko.flows.spline.NSF
+        elif model == 'ncsf':
+            logging.warning(
+                "You've selected a Neural Circular Spline Flow, for "
+                "which parameters are expected to be restricted to [-pi,pi]."
```

Comment:
Instead of just having a warning, can we catch such cases in the code in case they happen? Or does NCSF in zuko.flows.spline do this already?

Reply:
Thanks for catching this. It turns out zuko doesn't catch when theta's are out of range, it just tries to fit it anyway. I've added an error to catch this case, and a test to check for that error.